### DISCUSSION TITLE
Do you have any recommended Projects for real time AI Voice assistants

### DISCUSSION THREAD

**Discussion by GoudaCouda** [KoljaB/RealtimeTTS#241] [General]
I am looking for a real time Voice assistant tool someone has built that can run on local models. Obviously I could use the realtimeSTT and TTS together and an llm but im not great at coding

**Comment by KoljaB** [comment]
I've made https://github.com/KoljaB/LocalAIVoiceChat and https://github.com/KoljaB/LocalEmotionalAIVoiceChat, maybe you want to have a look into these, both run fully local.
